.travis.yml

```diff
@@ -36,15 +36,22 @@ jobs:
         - BUILD_WITH_ICC=true
       if: type = cron OR commit_message =~ /\[icc-build\]/
 
-    - python: 3.7
+    
```

Comment:
Is there any reason to add two blank spaces here?

Reply:
no :)